src/content/docs/Challenge/challenge-embedded.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Embedded Challenge
+description: Hardened JavaScript Escape Room on Embedded Hardware
+---
+
+<link rel="stylesheet" href="../index.css">
+
+The Embedded Challenge is a port of this site's [Web Challenge](https://hardenedjs.org/challenge/challenge) to [Moddable Six](https://www.moddable.com/moddable-six), an embedded device powered by the popular [ESP32-S3](https://www.espressif.com/en/products/socs/esp32-s3) microcontroller. The Embedded Challenge runs on the XS JavaScript engine. XS implements Hardened JavaScript natively, as part of the engine itself, including `lockdown()` and the `Compartment` class. The XS implementation of Hardened JavaScript is a completely separate implementation from the SES shim used by web-based Challenge.
+
+## Running the Embedded Challenge
+
+To run the Embedded Challenge, you'll need a Moddable Six (or equivalent) and a computer (macOS, Windows, or Linux) with the Moddable SDK and ESP-IDF installed. From there, it is straightforward to build, install, and run:
+
```

Comment:
Could use a tip on where to find the Moddable repo to clone and set up the MODDABLE environment variable.

Reply:
Good point. Will do.